src/dotnet/APIView/APIViewWeb/Models/ReviewModel.cs

```diff
@@ -75,6 +75,9 @@ public string DisplayName
         public DateTime LastUpdated => Revisions.LastOrDefault()?.CreationDate ?? CreationDate;
 
         [JsonIgnore]
-        public string Language => Revisions.FirstOrDefault()?.Files.FirstOrDefault()?.Language;
+        public string Language => Revisions.LastOrDefault()?.Files.LastOrDefault()?.Language;
+
+        [JsonIgnore]
+        public string PackageName => Revisions.LastOrDefault()?.Files.LastOrDefault()?.PackageName;
```

Comment:
Are these two used?

Reply:
Reverted this change. Not required anymore in current PR.